packages/x-date-pickers/src/PickersCalendarHeader/PickersCalendarHeader.tsx

```diff
@@ -183,8 +183,8 @@ const PickersCalendarHeader = React.forwardRef(function PickersCalendarHeader(
     className: classes.switchViewIcon,
   });
 
-  const selectNextMonth = () => onMonthChange(utils.addMonths(month, 1), 'left');
-  const selectPreviousMonth = () => onMonthChange(utils.addMonths(month, -1), 'right');
+  const selectNextMonth = () => onMonthChange(utils.addMonths(month, 1));
```

Comment:
Do you think I should count that as a breaking change?
Strictly speaking people can keep passing the param, it's just not used anymore and instead inferred by comparing the old and new month.